### PR TITLE
Design Updates

### LINKED ISSUE #28: ::selection is not noticeable
selecting text with a background so light, it's barely noticable

### PR DESCRIPTION
**Pull request by darcyclarke**
This should fix #28, #26 as well as continue to add nice subtitles to the design. 

/cc @jeffjewiss @richgilbank @rickycodes @brenna @Rchristiani @ahmadnassri @vladikoff @andrewdamelio

### CODE REVIEW

File: _sass/mentors.sass [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  .row
+    @extend .cf
+    display: block
+    clear: both
+
+  .row:nth-of-type(even)
+    @extend .cf
+    display: block
+    clear: both
+    transform: translate3d( 10%, 0, 0 )
```
Comment (Rchristiani) [inline _sass/mentors.sass:37]: I think `.list-attendees .row:nth-of-type(even)` should be `transform: translate3d(7%,0,0);` 10% is too much for the little attendees images.

**Comment by Rchristiani** [comment]
I think this looks good! Is is possible to get the images to load in a modal or something instead of a blank image? Other than that it looks nice! :+1:

**Comment by darcyclarke** [comment]
@Rchristiani I think a photo gallery is next up although I didn't feel like adding too much JS just yet. I'll modify the transformation though (as per your reco).

**Comment by Rchristiani** [comment]
:ok_hand:

**Comment by andrewdamelio** [comment]
:+1:

**Comment by chrismendis** [comment]
:+1:

**Comment by darcyclarke** [comment]
@Rchristiani fixed up that translation and set it to `7%`... @jeffjewiss can you merge? Or... anyone else...

**Comment by jeffjewiss** [comment]
@darcyclarke I think `tasks` needs to be added to the exclude section of `_config.yml` otherwise the tasks will get copied over to the site build. I'm good to merge after that one small change. Sorry for the delay!

**Comment by darcyclarke** [comment]
@jeffjewiss :+1: added it :D

**Comment by darcyclarke** [comment]
@nodeschool/toronto-mentors For future reference: when you make a pull request you can/should provide a link to your forked repo via Github Pages. It should look something like: `yourusername.github.io/toronto/` (ex. http://darcyclarke.github.io/toronto/). This way reviewers won't have to checkout a branch locally to run the site and see the changes :smile: Static site generators FTW

**Pull request merged**